R/polish_emissions_profile_company.R

```diff
@@ -1,37 +1,31 @@
-#' Creates final output of pctr company level results
+#' Creates final output of Emissions Profile company level results
 #'
-#' @description
-#' `r lifecycle::badge("deprecated")`
+#' @return A dataframe
 #'
-#' @inheritParams prepare_ictr_company
-#'
-#' @return A dataframe that prepares the final output of pctr_company
-#'
-#' @export
 #' @keywords internal
 #' @examples
 #' # See examples in `?profile_emissions`
-prepare_pctr_company <- function(pctr_comp, pctr_prod, comp, eco_activities, match_mapper, isic_tilt_map) {
-  pctr_prod <- sanitize_isic(pctr_prod)
+polish_emissions_profile_company <- function(ep_comp, ep_prod, europages_companies, ecoinvent_activities, ecoinvent_europages, isic) {
+  ep_prod <- sanitize_isic(ep_prod)
```

Comment:
@maurolepore I prefer to do both removing and refactoring in this PR because refactoring is not a big change as I can ensure it does not break anything. I created these functions in the beginning and very well understand the code. I hope it would be ok for you. Thanks for the review :)